examples/libtorchaudio/CMakeLists.txt

```diff
@@ -2,9 +2,6 @@ cmake_minimum_required(VERSION 3.5)
 
 project(libtorchaudio-cpp-example)
 
-SET(BUILD_SOX ON CACHE BOOL "Build libsox into libtorchaudio")
-
-SET(BUILD_KALDI OFF CACHE BOOL "Build Kaldi into libtorchaudio")
```

Comment:
While doing this, remove also the corresponding options from README.md and build.sh files as well.